src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Commands.xml

```diff
@@ -3,49 +3,82 @@
+      <Example Title="Add a source that needs authentication (then go install credential provider):" Command="dotnet nuget add source https://azureartifacts.microsoft.com/myTeam -n myTeam" />
+      <SeeAlso Title="Package source sections in NuGet.config files" Url="https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#package-source-sections" />
+      <SeeAlso Title="sources command (nuget.exe)" Url="https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-sources" />
     </Noun>
   </Verb>
   <Verb Name="disable">
-    <Noun Name="source">
+    <Noun Name="source" Version="3.1.200" 
+       Description="will disable an existing source in your NuGet configuration files."
+    >
       <Argument Name="name" Help="SourcesCommandNameDescription" />
       <SingleValueOption Name="configfile" Help="Option_ConfigFile"/>
+      <Example Title="Disable a source with name of `mySource`:" Command="dotnet nuget disable source mySource" />
+      <SeeAlso Title="Package source sections in NuGet.config files" Url="https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#package-source-sections" />
+      <SeeAlso Title="sources command (nuget.exe)" Url="https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-sources" />
     </Noun>
   </Verb>
   <Verb Name="enable">
-    <Noun Name="source">
+    <Noun Name="source" Version="3.1.200"
+       Description="will enable an existing source in your NuGet configuration files."
+    >
       <Argument Name="name" Help="SourcesCommandNameDescription" />
       <SingleValueOption Name="configfile" Help="Option_ConfigFile"/>
+      <Example Title="Enable a source with name of `mySource`:" Command="dotnet nuget enable source mySource" />
+      <SeeAlso Title="Package source sections in NuGet.config files" Url="https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#package-source-sections" />
+      <SeeAlso Title="sources command (nuget.exe)" Url="https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-sources" />
     </Noun>
   </Verb>
   <Verb Name="list">
-    <Noun Name="source">
+    <Noun Name="source" Version="3.1.200"
+       Description="will list all existing sources from your NuGet configuration files."
+    >
       <SingleValueOption Name="format" Help="SourcesCommandFormatDescription" />
       <SingleValueOption Name="configfile" Help="Option_ConfigFile"/>
+      <Example Title="List configured sources from the current directory:" Command="dotnet nuget list source" />
+      <SeeAlso Title="Package source sections in NuGet.config files" Url="https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#package-source-sections" />
+      <SeeAlso Title="sources command (nuget.exe)" Url="https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-sources" />
```

Comment:
If you are going to generate docs with these links included you need to generate different links not the culture specific ones: 

https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#links-to-articles-in-the-same-docset

Reply:
although links to docs within the same product should be relative to the .md file, so that the docs validator can validate the URLs are valid. If you use URLs, it won't check the link is valid.

Reply:
these docs will live in SDK docset...so we need full url...but i'm removing the locale from the links. thanks. good catch.

Reply:
I think the guidance is the not use the docs.microsoft.com part anyways.

Reply:
ah yes, i read Andy's link, but didn't read closely enough:
"For docsets that are also used in "offline books" (this includes Windows, .NET, ASP.NET, Visual Studio, C++, SQL Server, and NuGet)," ... use relative.

Reply:
fixed.